### ISSUE TITLE
FYI+FR: Add `localhost.direct` certificates

### ISSUE THREAD

**Issue by pataquets** [ddev/ddev#5466]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

The common "getting non-self-signed-valid-TLS-certificates-for-localhost"
It can be used for HTTP, MySQL, etc.

### Describe your solution

Discovered [localhost.direct](https://get.localhost.direct/) some time ago and found it very useful!

Just letting you know it's around, as I was a former ddev user. Surprisingly, it's not very popular, despite its usefulness.
Page says it all but, in summary, provides valid certs for `*.localhost.direct` + a `*.localhost.direct` DNS entry which resolves to `127.0.0.1`. Just that simple! :rocket: 

### Describe alternatives

Self-signed certs :disappointed:

**Comment by rfay** [comment]
Hi @pataquets - DDEV has had trusted certs for all projects for years - using mkcert. So everybody's normal experience is to use `https://<projectname>.ddev.site` and their browser trusts the cert. 

Can you say a bit about why you think this is relevant? DDEV also uses `*.ddev.site` by default, but can use any FQDN and browsers trust it. `*.ddev.site` all resolves to 127.0.0.1, just as localhost.direct does.

**Comment by pataquets** [comment]
It was some time ago when I last used ddev. Tried it out to refresh my memory and realized that there is still a little gain on using this:
- External dependency on `mkcert` (no big deal, just for completeness).
- You depend on modifying the user's host system to add certs to trust store (stores?). When testing, I had to manually run `mkcert -install` as it failed initially.
- Not working for other trust stores (e.g. Java apps).

Honestly, I had not a very recent recall of what was the local cert issue status, but assumed that a locally generated, self signed cert was being used (not too far from current status), but missed the addition to local trust store. Anyway, as I found the solution so simple and elegant, just wanted to share in case you find it useful and maybe an improvement.
Just my 2c.

In any case, feel free to close if it's not useful.
